README.md

```diff
@@ -27,11 +25,18 @@ Active Shipping is currently being used and improved in a production environment
 
 ## Installation
 
+Using bundler, add to the `Gemfile`:
+
 ```ruby
-gem install active_shipping
+gem 'active_shipping'
+```
+
+Or stand alone:
+
+```
+$ gem install active_shipping
 ```
 
-...or add it to your project's [Gemfile](http://bundler.io/).
 
 ## Sample Usage
```

Comment:
In here (well, below), can you update the `ActiveShipping::Package` comments to explain it's in HWL order? Nothing indicates that and having to dig into the source code to know something as trivial as parameter ordering kinda blows. Especially since we typically expect LWH in the shipping industry.